playwright/features/advSearch.feature

```diff
@@ -23,4 +23,14 @@ Feature: Advanced Search
         And I click the "advanced search button"
         Then the "keyword heading" should be displayed
         And the "author heading" should be displayed
-        And the "search result link" should be displayed
\ No newline at end of file
+        And the "search result link" should be displayed
+
+    Scenario: As a user I select a language checkbox in advance search and verify the search results
+        Given I go to the "home" page
+        When I click the "advanced search link"
+        Then I fill in the "keyword search box" with "Jane Austen"
+        And I click the "Russian language checkbox"
+        And I click the "advance page search button"
```

Comment:
can "advanced search button" from [mappings](https://github.com/NYPL/sfr-bookfinder-front-end/pull/424/files#diff-c18e09fe81c3095b5f131757f840255aab055b2d3f98e766eefb69a7970163beR26) be used here?